apps/E2E/src/RadioGroup/specs/RadioGroup.spec.win.ts

```diff
@@ -29,80 +28,133 @@ describe('RadioGroup/RadioButton Testing Initialization', function () {
-    );
+  it('Set RadioButton "accessibilityLabel" prop. Validate "accessibilityLabel" value propagates to "Name" element attribute.', async () => {
+    await expect(
+      await RadioGroupPageObject.compareAttribute(
+        RadioGroupPageObject.getRadioButton(RadioButton.First),
+        Attribute.AccessibilityLabel,
+        FIRST_RADIO_BUTTON_ACCESSIBILITY_LABEL,
+      ),
+    ).toBeTrue();
+
     await expect(await RadioGroupPageObject.didAssertPopup()).toBeFalsy(RadioGroupPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('RadioButton - Do not set accessibilityLabel -> Default to RadioButton label', async () => {
-    await expect(await RadioGroupPageObject.getRBAccessibilityLabel(RadioButtonSelector.Second)).toEqual(SECOND_RADIO_BUTTON_LABEL);
+  it('Do not set RadioButton "accessibilityLabel" prop. Validate "Name" element attribute defaults to current RadioButton label.', async () => {
+    await expect(
+      await RadioGroupPageObject.compareAttribute(
+        RadioGroupPageObject.getRadioButton(RadioButton.Second),
+        Attribute.AccessibilityLabel,
+        SECOND_RADIO_BUTTON_LABEL,
+      ),
+    ).toBeTrue();
+
     await expect(await RadioGroupPageObject.didAssertPopup()).toBeFalsy(RadioGroupPageObject.ERRORMESSAGE_ASSERT);
   });
 });
 
 describe('RadioGroup Functional Testing', async () => {
   /* This resets the RadioGroup state by clicking/selecting the 1st RadioButton in the RadioGroup */
   beforeEach(async () => {
-    await RadioGroupPageObject.scrollToTestElement();
+    await RadioGroupPageObject.scrollToTestElement(await RadioGroupPageObject._firstRadioGroup);
 
     await RadioGroupPageObject.resetRadioGroupSelection();
   });
 
-  it('Click on a RadioButton and ensure it changes state from unselected -> selected', async () => {
+  it('Click on a RadioButton. Validate that it changes state from unselected to selected.', async () => {
     /* Validate the RadioButton is not initially selected */
-    await expect(await RadioGroupPageObject.isRadioButtonSelected(RadioButtonSelector.Second)).toBeFalsy();
+    await expect(await RadioGroupPageObject.isRadioButtonSelected(RadioButton.Second)).toBeFalsy(
+      'Expected the first RadioButton to be initially selected, but the second RadioButton was initially selected.',
+    );
 
     /* Click on the RadioButton to select it */
-    await RadioGroupPageObject.clickRadioButton(RadioButtonSelector.Second);
-    await RadioGroupPageObject.waitForRadioButtonSelected(RadioButtonSelector.Second, PAGE_TIMEOUT);
+    await RadioGroupPageObject.click(RadioGroupPageObject.getRadioButton(RadioButton.Second));
 
     /* Validate the RadioButton is selected */
-    await expect(await RadioGroupPageObject.isRadioButtonSelected(RadioButtonSelector.Second)).toBeTruthy();
+    await expect(
+      await RadioGroupPageObject.waitForRadioButtonSelected(
+        RadioButton.Second,
+        'Clicked the second RadioButton, but it failed to be selected.',
+      ),
+    ).toBeTruthy();
```

Comment:
Using `.toBeTruthy()` here and `.toBeTrue()` above. We should use the same one throughout the repo.

Is there a functional difference?

Reply:
"Truthy-ness", if I recall, encompasses values such as true booleans, numbers > 0 and non-empty strings whereas True only means a true boolean. Since this returns true on success, both will pass, though we should indeed stick to `.toBeTrue()`.

Reply:
On closer inspection, `.toBeTruthy()` accepts an optional parameter that allows you to throw an error message if the truthy test fails. Maybe sticking to `.toBeTruthy` might be the better move.